pydoctor/themes/base/apidocs.css

```
@@ -371,6 +371,12 @@ code > a, #childList > div .functionHeader a {
     color:#c7254e;
     background-color:#f9f2f4;
 }
+
+/* Intersphinx links are not red, but simply blue */
+a.intersphinx-link {
+    color:#306998!important;
```

very minor suggestion

at least on my computer, with the grey background, this blue has little contrast and saturation. and looks like dark grey

```suggestion
    color:##06f!important;
```